docs/guides/formulasC2/build-template.md

```diff
@@ -0,0 +1,260 @@
+</a>" type="info" %}
+
+## Build a Formula Template
+
+Formula templates are comprised of a trigger that kicks off the formula, steps that execute as a result of the trigger, and variables to represent input required to run an instance of the formula. You can create formulas that use triggers that are kicked off when something happens to an element instance, you can schedule triggers, or you can manually kick off a trigger. See [Triggers, Steps, and Variables](reference.html) for more information about each component.
+
+As you build a formula, you also build context consisting of triggers, variables, steps, and values created by steps. You can refer to that context in later steps in the formula. Consider each component as a building block that has a name and information that you can refer to. For example, in a formula that sends an email notification for a new contact, you build the body of the email in one step. In a later step when you actually send the message, refer to the step where you built the body. See [Reference Syntax](#reference-syntax) for ways to refer to triggers.
+
+{% include note.html content="These steps are intended to get you started building a formula template. Consider using some of <a href=examples.html>the example formulas</a> to get a deeper understanding of how to build a formula template.  " %}
+
+To build a new formula template:
+
+1. Click **Formulas**, and then on the Formulas page, click **Create New Formula**.
+![New Formula Template](img/new-formula.png)
+3. Click **Create New Formula**.
+4. Enter a name for your formula, and then click **Create**.
+
+    {{site.console}} provides you a list of triggers. For details about each trigger, see [Triggers, Steps, and Variables](reference.html).
+    ![Triggers](img/triggers.png)
+
+5. Select your trigger.
+    * Choose **Event** for a formula triggered by an event configured on an Element.
+    * Choose **Element Request** for a formula triggered when a specific API call is made to an element.
+    * Choose **Scheduled** for a formula to occur at a specific time or regular interval.
+    * Chose **Manual** to trigger the formula with an API call to `POST /formulas/instances/:id/executions`.
+
+    {% include note.html content="You cannot change a trigger in a formula template after you create it.  " %}
```

Comment:
@davewilks it just moves them off screen, the steps are still saved in that created but unused section.